js/src/dapps/registry/accounts/accounts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+
+// import styles from './lookup.css';
+
+const renderAccount = (active) => (account) => (
```

Comment:
> The code is react-only anyway, that function is returning a JSX tree.

JSX is just syntax. In *this specific case*, it will be transpiled into reactdom calls. So the function returns a React-specific virtual tree, but could easily return DOM or virtual-dom.

I know this is a general discussion wether to go further down the React road or keep it more "open-minded". Let's discuss this in Slack.